evap/staff/tests/test_views.py

```diff
@@ -1219,6 +1219,11 @@ def setUpTestData(cls):
         cls.manager_user = baker.make(UserProfile, username='manager', groups=[Group.objects.get(name='Manager')])
         cls.course = baker.make(Course, semester=baker.make(Semester, pk=1))
 
+    def test_course_is_prefilled(self):
+        response = self.app.get(f'{self.url}/{self.course.pk}', user='manager', status=200)
+        form = response.context['form']
```

Comment:
This is something I found today: You can retrieve the Django objects (like the form) via `response.context`. There even exists a [`self.assertFormError`](https://docs.djangoproject.com/en/3.0/topics/testing/tools/#django.test.SimpleTestCase.assertFormError), which may be the thing I looked for. Anyway, in this test case I queried the django form just for simple convenience as the webtest form would require to put `str(self.course.pk)` on the right side (so no big advantage here)